packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard/signup-wizard.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { Wizard } from '@deriv/ui';
+
+type TSignupWizardProps = {
+    setIsWizardOpen: (is_wizard_open: boolean) => void;
```

Comment:
@george-usynin-binary It's not a good idea to make this component aware of outside, I suggest having callbacks like `onOpen` or `onClose` and letting the parent component handle the logic.

Reply:
@farzin-deriv agree with you, will refactor it